src/lib.rs

```diff
@@ -1,3 +1,7 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
```

Comment:
In other places we do this like:
```
#[cfg(not(feature = "std"))]
#[macro_use]
extern crate alloc;
```
Should we do it the same way here to keep things consistent?

Reply:
I had to do it slightly different because these macros are currently used only in tests. We achieve the desired result with the cfg_attr